_posts/2015-12-11-License-and-You.markdown

```diff
@@ -0,0 +1,123 @@
+
+Recommendation for further reading:
+
+* http://choosealicense.com
+* https://tldrlegal.com/
+
+We'll talk about 3 types of licenses:
+
+- Permissive: MIT/BSD, used by jQuery, Rails, Artsy OSS
+- Mostly Permissive: Apache/Mozzila, used by SVN, Firefox
+- Copyleft: GPL, used by Git, Linux, Wordpress
+
+### Common Components
+
+All share one important thing:
+
+  * YOU MAY USE THIS
+  * NO WARRANTY
+  * DISCLAIMER OF LIABILITY
+
+Which is basically the crux of how we can all actually use and write OSS without ending up bankrupt. E.g. when a major component of your infrastructure "fails" - it's not the fault of the author. Example: [Shellshock](https://en.wikipedia.org/wiki/Shellshock_(software_bug)) - no-one tried suing the maintainers of Bash.
+
+### License Free
+
+By choosing to not include a license, you only applying your copyright to the code. All code is the copyright of someone, because creation of anything is covered by copyright. Copyright is, err, complex. Roughly speaking though, there is no contract between you and someone using your code. This means _all_ of the power is in favour of the library author. A library consumer would need to email the author and ask for permission on a case by case basis, even then there is nothing stopping a developer revoking the ability for you to use their library after the fact. It's also different depending on the country the library author is in. However in the end the default copyright is not a license, so add one.
+
+
+[This is basically the default license for code. ](http://blog.codinghorror.com/pick-a-license-any-license/)
+
+This means that code on a blog could not be safe to use unless they've declared its license, whereas for something like stack overflow all of the contents there are licensed under the Creative Commons. Which I'll talk about later.
+
+Just uploading some code to GitHub actually does put it under a license of sort, which includes the ability for someone to fork and view the code. It's explicit in the GitHub T&C, they could not have those features without it. Other than that though, you offer no contract with the library user.
+
+### OSI
+
+So when we say Open Source license, we are talking about licenses that come from the Open Source Initiative. They are a pragmatic group who say whether a license should be classed as 'open source' or not. A bit weird, but they generally just debate a lot of the legal stuff in these licenses and try to find holes so that everyone is protected. It's basically a seal of approval. There are very few licenses that are not OSI approved, the one you might know is the WTFPL - which is basically default copyright anyway.
+
+On the mobile team we requested that dependencies of ours convert from WTFPL so that we don’t have the issue mention about the public domain, we as Artsy have little protection against the library authors future wishes.
```

Comment:
Here you refer to it as 'public domain', but you changed that naming earlier in the post to 'license free'.